data/json/recipes/recipe_food.json

```diff
@@ -7693,6 +7693,18 @@
   ]
 },{
   "type" : "recipe",
+  "result": "vodka",
```

Comment:
Vodka *is* ethanol + water; high-quality vodka contains absolutely nothing else.